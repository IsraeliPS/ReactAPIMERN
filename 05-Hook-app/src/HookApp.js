import React from 'react'

function HookApp () {
  return (
    <div>
      <h1>
        HookApp
      </h1>
    </div>
  )
}

export default HookApp
