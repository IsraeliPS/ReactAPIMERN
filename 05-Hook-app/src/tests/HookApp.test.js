import React from 'react'
import { shallow } from 'enzyme'
import '@testing-library/jest-dom'
import '../setupTests'

import HookApp from '../HookApp'

describe('Pruebas en <HookApp />', () => {
  test('Debe mostrar correctamente el componente', () => {
    const wrapper = shallow(<HookApp />)
    expect(wrapper).toMatchSnapshot()
  })
})
