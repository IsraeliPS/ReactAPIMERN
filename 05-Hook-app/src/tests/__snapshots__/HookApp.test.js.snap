// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Debe mostrar correctamente el componente 1`] = `
<div>
  <h1>
    HookApp
  </h1>
</div>
`;
